server/api/sample_sizes.py

```diff
@@ -214,6 +213,20 @@ def next_round_sample_size_options(election_id: str):
     sample_sizes.sample_size_options = sample_size_options(election)
 
 
+# In rounds other than the first round, we want to automatically select a sample
+# size from the generated options instead of letting the user pick.
```

Comment:
Just curious why we let users pick from a set of options for the first round but not for later rounds. Is it because, if we can't complete the audit in one round, we have to start getting more prescriptive in subsequent rounds to ensure that the audit completes?

Reply:
I'm not sure why to be honest